examples/plotting/text.py

```diff
@@ -1,3 +1,12 @@
+'''This example shows multiple variations in angle and alignment to multi-line text using CustomJS.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.model.CustomJS, bokeh.model.Slider
+    :refs: :ref:`bokeh.models.Slider.js_on_change`
```

Comment:
We try to link to the user's guide where possible
```suggestion
    :refs: :ref:`ug_interaction_js_callbacks_customjs`
```